kore/src/Kore/Log/DebugTransition.hs

```diff
@@ -0,0 +1,117 @@
+{- |
+Copyright   : (c) Runtime Verification, 2020
```

Comment:
```suggestion
Copyright   : (c) Runtime Verification, 2021
```

Reply:
I suppose this is far enough away from the old `DebugClaimState` that this is warranted, done.